.travis.yml

```diff
@@ -126,4 +127,4 @@ script:
 #      bin/deploy_doc.sh;
 #    fi
 notifications:
-  email: false
+  email: true
```

Comment:
Please don't change this. This will send email notifications to all of the core developers, which is undesirable.